app/assets/stylesheets/components/_border.scss

```diff
@@ -10,6 +10,10 @@
 .rounded-xl { border-radius: $border-radius-xl; }
 .rounded-xxl { border-radius: $border-radius-xxl; }
 
+.border-blue-medium {
```

Comment:
Yeah if the color of the borders have changed then we should update it to whatever @rtwell specified, but we should just use the existing `$border-color` so it updates across the board.